packages/connection-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-certificate-authority.tsx

```diff
@@ -26,20 +38,38 @@ function TLSCertificateAuthority({
     <FormFieldContainer className={caFieldsContainer}>
       <FileInput
         description={'Learn More'}
-        disabled={disabled}
+        disabled={disabled || useSystemCA}
```

Comment:
Just a question about this, are those strict alternatives? or is theoretically possible to pass part of the CA chain as a file and have other certs for parent CAs in a system folder? Nothing to address anyway, I'm just curious.

Reply:
As soon as one certificate in the chain is part of the CA, the server’s certificate will be considered trusted – it wouldn’t really make sense to specify two separate lists of trusted CAs for a single endpoint.

Additionally, the way that the Node.js driver currently works is that it reads the `tlsCAFile` and stores it in the `ca` option which `useSystemCA` maps to, so they would conflict in the Node.js driver options parsing. (Plus, we should keep the liberty to eventually implement `useSystemCA` via `tlsCAFile` ourselves as well.)